packages/schema/src/config/vite.ts

```diff
@@ -92,14 +92,14 @@ export default defineUntypedSchema({
         allow: {
           $resolve: async (val: string[] | undefined, get) => {
             const [buildDir, srcDir, rootDir, workspaceDir, modulesDir] = await Promise.all([get('buildDir'), get('srcDir'), get('rootDir'), get('workspaceDir'), get('modulesDir')]) as [string, string, string, string, string]
-            return [
+            return [...new Set([
```

Comment:
Just curious was it making a bug or just is for docs?

Reply:
For the docs mostly but I did see a related vite issue where adding extra entries to an array increased startup time disproportionately so thought it was worth resolving.